# This is actually the approach point for tiago, I need to change the name for this so that is either actually is relative to the girl_pointing or change the name to approach_point
girl_pointing:
  position: 
    x: -2.456
    y: -6.697
    z: 0.0985
  orientation: 
    x: 0.0
    y: 0.0
    z: -0.586
    w: 0.810


# This is calculated by finding the center point of the table, which is pose in gazebo, and then the pose of the links gives us the relative position of each leg to the center, we just need to calculate it
tables:
  - name: 'Table 1'
    bottom_left: { x: -1.63, y: -8.54, z: 0.0 }
    bottom_right: { x: -2.59, y: -8.54, z: 0.0 }
    top_left: { x: -1.63, y: -9.30, z: 0.0 }
    top_right: { x: -2.59, y: -9.30, z: 0.0 }

  - name: 'Table 2'
    bottom_left: { x: 0.81, y: -10.77, z: 0.0 }
    bottom_right: { x: -0.54808, y: -10.77, z: 0.0 }
    top_left: { x: 0.81, y: -11.53, z: 0.0 }
    top_right: { x: -0.54808, y: -11.53, z: 0.0 }

  - name: 'Table 3'
    bottom_left: { x: 3.00, y: -8.23, z: 0.0 }
    bottom_right: { x: 2.04, y: -8.23, z: 0.0 }
    top_left: { x: 3.00, y: -8.99, z: 0.0 }
    top_right: { x: 2.04, y: -8.99, z: 0.0 }

  - name: 'Table 4'
    bottom_left: { x: 3.65, y: -5.10, z: 0.0 }
    bottom_right: { x: 2.69, y: -5.10, z: 0.0 }
    top_left: { x: 3.65, y: -5.86, z: 0.0 }
    top_right: { x: 2.69, y: -5.86, z: 0.0 }
